tmt/cli.py

```diff
@@ -1623,3 +1623,62 @@ def completion_fish(context: Context, install: bool, **kwargs: Any) -> None:
     Setup shell completions for fish.
     """
     setup_completion('fish', install)
+
+
+@click.command(context_settings=dict(
+    ignore_unknown_options=True,
+    allow_extra_args=True,
+    ))
+@click.pass_context
+@click.option(
+    '-r', '--root', metavar='PATH', show_default=True, default='.',
+    help='Variable passed to tmt main. See `tmt --help`')
+@click.option(
+    '-v', '--verbose', count=True, default=0,
+    help='Variable passed to tmt main. See `tmt --help`')
+@click.option(
+    '--version', is_flag=True,
+    help="Variable passed to tmt main. See `tmt --help`")
+@click.option(
+    '--lint-type', default=None,
```

Comment:
`--lint-type` sounds a bit too technological, strict. Maybe something common-ish, like `--lint` or `--lint-what`? As in "lint what, tests? or plans? `--lint-what tests` would IMHO be easier to grasp for a common user.

Reply:
I think `--lint` is a bit confusing that it might be a flag instead of an option. I am inclined more to `--lint-what`, but there's something subconsciously that feels off. Maybe it is the grammar flow of it? I wonder if there is something more appropriate to all. Something that can indicate a set/object/group.

Reply:
@lukaszachy do you have a preference or suggestions on the naming?